charts/zeebe-benchmark/values-realistic-benchmark.yaml

```diff
@@ -217,69 +213,105 @@ starter:
+        value: "http://pyroscope.pyroscope.svc.cluster.local:4040"
+      - name: PYROSCOPE_APPLICATION_NAME
+        value: "io.camunda.zeebe.broker"
+      - name: PYROSCOPE_LOG_LEVEL
+        value: "debug"
+      - name: PYROSCOPE_FORMAT
+        value: "jfr"
+      - name: PYROSCOPE_PROFILER_EVENT
+        value: "cpu"
+      - name: PYROSCOPE_PROFILER_ALLOC
+        value: "0"
+      - name: PYROSCOPE_PROFILER_LOCK
+        value: "0"
+      - name: PYROSCOPE_LABELS
+        value: "namespace=$(K8S_NAMESPACE), pod=$(K8S_NAME)"
+      # To be able to load a separate/additional configuration
+      # See https://github.com/camunda/camunda-platform-helm/issues/2197
+      - name: SPRING_CONFIG_ADDITIONALLOCATION
+        value: "/usr/local/camunda/config/benchmark-config.yaml"
+      - name: JAVA_OPTS
+        valueFrom:
+          configMapKeyRef:
+            name: zeebe-config
+            key: java-opts
+            optional: true
+      # To run Operate and Tasklist add these to the value of this variable
+      - name: SPRING_PROFILES_ACTIVE
+        value: "auth,broker,operate"
 
-  zeebeGateway:
-    # Replicas defines how many standalone gateways are deployed
-    replicas: 2
+    ## @param core.debug if true, extra info is printed.
+    debug: false
 
-    # Image configuration to configure the zeebe-gateway image specifics
+    ## @extra core.image configuration to configure the image specifics
     image:
-      # Image.repository defines which image repository to use
-      repository: camunda/zeebe
+      ## @param core.image.repository defines which image repository to use
+      repository: camunda/camunda
+      ## @param core.image.tag can be set to overwrite the global tag, which should be used in that chart
       tag: SNAPSHOT
 
-    # Resources configuration to set request and limit configuration for the container https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits
-    resources:
-      limits:
-        cpu: 450m
-        memory: 1Gi
-      requests:
-        cpu: 450m
-        memory: 1Gi
 
-  operate:
-    enabled: true
-    image:
-      repository: camunda/operate
-      tag: SNAPSHOT
-    # Resources configuration to set request and limit configuration for the container https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits
+    ## @param core.logLevel defines the log level which is used
+    logLevel: info
+
+    ## @extra core.resources configuration to set request and limit configuration for the container https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits
+    ## @extra core.resources.requests
+    ## @param core.resources.requests.cpu
+    ## @param core.resources.requests.memory
+    ## @param core.resources.limits.cpu
+    ## @param core.resources.limits.memory
     resources:
-      requests:
-        cpu: 600m
-        memory: 400Mi
       limits:
+        cpu: 3500m
```

Comment:
We need to check whether this fits in our nodes

Reply:
I changed the node selector to a bigger one 😄